src/macaron/policy_engine/prelude/aggregate_rules.dl

```diff
@@ -0,0 +1,59 @@
+/* Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved. */
+/* Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/. */
+
+
+/* aggregate verifying a level */
+.decl aggregate_level(level: number, repo: number)
+.decl aggregate_level_requirement(level: number, repo: number)
+
+.decl agg_levels(n: number)
+agg_levels(0).
+agg_levels(n+1) :- n <= 4, agg_levels(n).
+
+/**
+ * Everything has a repository and uses a scripted build service.
+ */
+aggregate_level_requirement(1, repo) :-
+    repository(repo, _,_,_,_,_,_),
+    check_passed(repo, "mcn_build_service_1"),
+    check_passed(repo, "mcn_version_control_system_1").
+
+
+/**
+  * The build is verifiably automated and .
+  */
+aggregate_level_requirement(2, repo) :-
+    repository(repo, _,_,_,_,_,_),
+    check_passed(repo, "mcn_build_script_1"),
+    check_passed(repo, "mcn_build_service_1"),
+    check_passed(repo, "mcn_build_as_code_1").
+
+
+// The minimum aggregate level a reposotories dependencies have
```

Comment:
```suggestion
 \**
  * The minimum aggregate level the dependencies of a repository should have.
  */
```

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/46/commits/7584d191a925261dd109b81512af671aa81d42aa